src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -47,40 +45,8 @@
   RawContestConfig() {
   }
 
-  // function: validate
-  // purpose: perform some basic validation checks
-  // returns: true if checks pass otherwise false
-  public boolean validate() {
```

Comment:
Confirming my understanding: you removed this method because Jackson will now automatically throw an exception if any of these fields are missing?

Reply:
this was never called, I think Hylton removed the calls in an earlier code change.